spec/team_spec.rb

```diff
@@ -5,8 +5,8 @@
   it 'exists and has readable attributes' do
     info = {
       team_id: "1",
-      franchiseId: "23",
-      teamName: "Atlanta United",
```

Comment:
nice catch on changing the symbols to all lower case. i still forget this time to time